include/myst/thread.h

```diff
@@ -88,6 +88,9 @@ struct myst_thread
     /* unique thread identifier (same as pid for main thread) */
     pid_t tid;
 
+    /* the value returned by gettid() on the target (for this thread) */
```

Comment:
`myst_gettid()` still returns the internal `tid` above. `myst_sched_getaffinity()` and myst_sched_setaffinity()' perform the internal-tid to `target_tid` translation. The comment should be fixed.